tls-core/src/lib.rs

```diff
@@ -1,2 +1,3 @@
 #[cfg(feature = "prf")]
 pub mod prf;
+pub mod ghash;
```

Comment:
Let's add a feature gate for the ghash module, same as how I had the prf module configured.

Also we should add a newline at the end of this file